addon/components/nrg-application/component.js

```diff
@@ -56,5 +56,10 @@ export default Component.extend(ResizeMixin, {
         this.toggleProperty('sidebarIsOpen');
       }
     },
+    clickedLink(item) {
+      if (this.clicked) {
+        this.clicked(item);
```

Comment:
What are your thoughts on switching the names of these two?

"clicked" doesn't really showcase the use case of the action. What about `clickedSidebarItem`?